airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/context.py

```diff
@@ -286,3 +295,12 @@ async def __aexit__(
 
     def create_slack_message(self) -> str:
         raise NotImplementedError
+
+    def _get_step_id_to_skip_according_to_metadata(self) -> List[CONNECTOR_TEST_STEP_ID]:
```

Comment:
📚 We need a doc string here explaining the what and why.

I knew what were doing but even I struggled reading the list comprehension